apps/cookbook/src/app/examples/button-example/examples/attention-level.scss

```diff
@@ -0,0 +1,23 @@
+@use 'sass:map';
+@use '@kirbydesign/core/src/scss/utils';
+
+@use './shared';
+
+:host {
+  display: flex;
+  flex-direction: row;
```

Comment:
`row` is the initial value for `flex-direction` so you probably don't need this declaration